spec/helpers/mock_project.rb

```diff
@@ -119,14 +119,37 @@ def bundle_exec(command, env = {})
       end
     end
 
+    TAPIOCA_DEPENDENCIES = T.let(
+      [
+        "diff-lcs",
+        "rbi",
+        "spoom",
+        "tapioca",
+        "thor",
+        "unparser",
+        "yard",
+        "yard-sorbet",
+      ],
+      T::Array[String],
+    )
+
     # Run a Tapioca `command` with `bundle exec` in this project context (unbundled env)
-    sig { params(command: String, enforce_typechecking: T::Boolean).returns(ExecResult) }
-    def tapioca(command, enforce_typechecking: true)
+    sig do
+      params(
+        command: String,
+        enforce_typechecking: T::Boolean,
+        exclude_tapioca_dependencies: T::Boolean,
+      ).returns(ExecResult)
+    end
+    def tapioca(command, enforce_typechecking: true, exclude_tapioca_dependencies: true)
       exec_command = ["tapioca", command]
       if command.start_with?(/gem/)
         exec_command << "--workers=1" unless command.match?("--workers")
         exec_command << "--no-doc" unless command.match?("--doc")
         exec_command << "--no-loc" unless command.match?("--loc")
+        if exclude_tapioca_dependencies && !command.match?("--exclude")
+          exec_command << "--exclude #{TAPIOCA_DEPENDENCIES.join(" ")}"
+        end
```

Comment:
I didn't want to complicate this but ideally we'd append the arguments to `exclude` flag with the `TAPIOCA_DEPENDENCIES` and enable this speedup on tests that use `exclude` too.